monitor/data_monitoring.py

```diff
@@ -152,6 +169,21 @@ def generate_report(self) -> bool:
             self.execution_properties['report_end'] = True
             self.execution_properties['success'] = self.success
             return self.success
+    
+    def send_report(self) -> bool:
+        elementary_html_file_path = os.path.join(self.config.target_dir, 'elementary.html')
+        if os.path.exists(elementary_html_file_path):
+            file_uploaded_succesfully = self.slack_client.upload_file(
+                channel_name=self.slack_channel_name,
+                file_path=elementary_html_file_path,
+                message=SlackMessageSchema(text="Elementary monitoring report")
+            )
+            if not file_uploaded_succesfully:
+                self.success = False
+        else:
+            logger.error('Could not send "Elementary monitor report" because it is not exists.')
```

Comment:
cosmetics - 'Could not send Elementary monitoring report because it does not exist'